cpu/esp8266/Makefile.include

```diff
@@ -0,0 +1,165 @@
+# check some environment variables first
+ifndef NEWLIB_DIR
+    $(info NEWLIB_DIR should be defined as /path/to/newlib directory)
+    $(info NEWLIB_DIR is set by default to /opt/esp/newlib-xtensa)
+    export NEWLIB_DIR=/opt/esp/newlib-xtensa
+endif
+
+ifndef SDK_DIR
```

Comment:
plese change to ESP_SDK_DIR

(for both variables: or ESP8266_... if ESP32 needs another toolchain)

Reply:
Done